mappings/net/minecraft/entity/ai/brain/Brain.mapping

```diff
@@ -20,3 +25,4 @@ CLASS aiu net/minecraft/entity/ai/brain/Brain
 	METHOD b stopAllTasks (Lvc;Laif;)V
 	METHOD c getMemory (Lanw;)Ljava/util/Optional;
 	METHOD c hasActivity (Lavw;)Z
+	METHOD e clone ()Laiu;
```

Comment:
Are you kidding me... This is probably the worst mapping that can ever happen.

Reply:
elaborate?

Reply:
Looks like it overrides `Object#clone` by accident. #546

Reply:
You’re probably right, but curious: if Mojang did actually override clone() would it be obfuscated? Do we have a reliable way to know if this is or isn’t clone()?

Reply:
Oops. 😄 
Apologies about that..

Reply:
It shouldn't be. I think now enigma correctly handles synthetic methods generated for such overrides that changes the return type in the signature (see metadata section reader classes)